packages/db/src/core/load-file.ts

```diff
@@ -0,0 +1,141 @@
+import { build as esbuild } from 'esbuild';
+import { CONFIG_FILE_NAMES, VIRTUAL_MODULE_ID } from './consts.js';
+import { fileURLToPath } from 'node:url';
+import {
+	getConfigVirtualModContents,
+	getStudioVirtualModContents,
+} from './integration/vite-plugin-db.js';
+import type { DBTables } from './types.js';
+import { writeFile, unlink } from 'node:fs/promises';
+import { getDbDirUrl } from './utils.js';
+import { existsSync } from 'node:fs';
+
+export async function executeFile({
```

Comment:
nit, but it's unclear what kind of file we're talking about here. Is this for loading the `db/config.ts` file? `db/seed.ts`? A custom seed file for `astro db execute`? 

Renaming the file to be less generic might help, or at the very least adding a comment to the file or the individual functions.

Reply:
This file hold the general logic to load a JS file. There are two domain-specific functions:
- executeFile for `db execute`
- `loadConfigFile` for `db/config.ts`

I can move these domain-specific functions out of this file for clarity, or add jsdoc

Reply:
Update: moved executeFile() logic inline with the execute cli command. Also renamed `loadConfigFile()` to `loadDbConfigFile()`